deployer/deployer-services/src/main/java/io/hyscale/deployer/services/deployer/Deployer.java

```diff
@@ -108,6 +108,20 @@ public default void waitForDeployment(DeploymentContext context, ProgressHandler
 	public List<ReplicaInfo> getReplicas(AuthConfig authConfig, String appName, String serviceName, String namespace, 
 	        boolean isFilter) throws HyscaleException;
 	
+	
+	
+	/**
+	 * Exec into  Pods
+	 * @param k8sAuthorisation
+	 * @param servicename
+	 * @param appname
+	 * @param namespace
+	 * @param replica
+	 * @return exitCode
+	 * @throws HyscaleException
+	 */
+	 public int exec(K8sAuthorisation k8sAuthorisation, String servicename, String appname, String namespace, String replica) throws HyscaleException;
```

Comment:
Would we need the return type in case of platform?
Also camel case naming convention
Also name the variable order: K8sAuth, appName, ServiceName, replicaName, namespace